Add custom resource policy flag to the enable module command (#1584)
kyma-project/cli: docs/gen-docs/kyma_alpha_enable_module.md

  -f, --force-conflicts    Forces the patching of Kyma spec modules in case their managed field was edited by a source other than Kyma CLI.
  -k, --kyma-name string   Kyma resource to use. If empty, 'default-kyma' is used. (default "default-kyma")
  -n, --namespace string   Kyma Namespace to use. If empty, the default 'kyma-system' Namespace is used. (default "kyma-system")

>   -p, --policy string      Determines how the module is managed, either install or ignore the default values provided in the module template. Available values are 'CreateAndDelete' or 'Ignore'. (default "CreateAndDelete")

Reviewer: ```suggestion
  -p, --policy string      Determines how the module is managed. Use 'CreateAndDelete' to install the default values provided in the module template or 'Ignore' to ignore them. (default "CreateAndDelete")
```

Author: Changed.
